cdk/lib/__snapshots__/prism.test.ts.snap

```diff
@@ -785,6 +816,55 @@ dpkg -i /prism/prism.deb",
       },
       "Type": "AWS::IAM::Policy",
     },
+    "WazuhSecurityGroup": Object {
```

Comment:
I'm a little surprised we didn't have Wazuh already. IIUC the agent is installed on the AMI by default in AMIgo now.

cc @jorgeazevedo @philmcmahon @Nirvikalpa108